webroot/src/pages/Login/Login.ts

```diff
@@ -58,10 +61,44 @@ export default class Login extends Vue {
         return loginUri;
     }
 
+    get isUsingMockApi(): boolean {
+        return this.$envConfig.isUsingMockApi || false;
+    }
+
     //
     // Methods
     //
     redirectToHostedLogin(): void {
         window.location.replace(this.hostedLoginUriStaff);
     }
+
+    async mockStaffLogin(): Promise<void> {
+        const data = {
+            access_token: 'sjdkfslkjfhsdkjhfdskj',
+            token_type: 'Bearer',
+            expires_in: '100000000',
+            id_token: 'idToken',
+            refresh_token: 'refreshToken'
+        };
+
+        await this.$store.dispatch('user/storeAuthTokens', { tokenResponse: data, authType: AuthTypes.STAFF });
+        this.$store.dispatch('user/loginSuccess');
+
+        this.$router.push({ name: 'Home' });
+    }
+
+    async mockLicenseeLogin(): Promise<void> {
+        const data = {
+            access_token: 'dflkjdfhwhefjhkfdkfshdlkjfs',
+            token_type: 'Bearer',
+            expires_in: '100000000',
+            id_token: 'idToken',
+            refresh_token: 'refreshToken'
```

Comment:
Minor: Would it be better if these token mock values were simpler and also easily confirmed as mock values and also the user type? E.g. `mock_staff_access_token`, `mock_licensee_id_token`, etc?